app/demo-geant-integration/RunAction.cc

```diff
@@ -62,8 +62,9 @@ void RunAction::BeginOfRunAction(G4Run const* run)
                 = GlobalSetup::Instance()->GetGeometryFile();
         }
 
-        // Create the along-step action
-        GlobalSetup::Instance()->SetAlongStep(NoFieldAlongStepFactory{});
+        // Create the along-step action (no field)
```

Comment:
```suggestion
        // Create the along-step action with a uniform magnetic field (zero field by default)
```

Reply:
It is using the constructor specifically for not having a field, though...